contracts/UnipoolGIVpower.sol

```diff
@@ -36,7 +35,7 @@ contract UnipoolGIVpower is UnipoolTokenDistributor, IERC20MetadataUpgradeable {
     /// @notice Mapping with all accounts have locked tokens
     mapping(address => UserLock) public userLocks;
 
-    mapping(address => uint256) public uniBalances;
+    mapping(address => uint256) public depositTokenBalance;
```

Comment:
This variable is not needed. You can change [LPTokenWrapper _balances](https://github.com/Giveth/givpower/blob/9b02bddc927f1e3a58efe098c78a990c3361815d/contracts/UnipoolTokenDistributor.sol#L29)'s visibility to make it an internal variable, and you have the same values.

Reply:
These two have different values. Consider this scenario, user stake 100 GIV and lock it for on round, so it's power will be around 141
_balances[<user address] -> 141
depositTokenBalance[<user address] -> 100

Reply:
Oh sure! I didn't remember that we had to keep track of both amounts, because we have to check that you can't lock more than what you already staked. These contracts are complicated! I'll do a final review and I think it should be good to go!

Reply:
@sembrestels Did you get a chance to finish the final review?